### PR TITLE
Add server commands to chat auto completion word list

### LINKED ISSUE #535: Add commands to the chat autocompletion word list
It would be nice to have autocompletion of commands in the chat line. This can be achieved by simply adding them to the word list:

https://github.com/longturn/freeciv21/blob/80d2c1546a2e3b7398daa7dd677b2ef54f6d1d8c/client/gui-qt/chatline.cpp#L54

Make sure to have the `/` so they don't randomly appear in the text.

### PR DESCRIPTION
**Pull request by jwrober**
Closes #535

### CODE REVIEW

File: client/chatline.cpp [hunk 1]
```diff
@@ -385,7 +577,7 @@ void chat_widget::set_chat_visible(bool visible)
 }
 
 /**
- * Shows the chat and ensures the chat line has focus
+   Shows the chat and ensures the chat line has focus
```
Comment (lmoureaux) [inline client/chatline.cpp:580]: The * was correct :D
```suggestion
 * Shows the chat and ensures the chat line has focus
```

**Review by lmoureaux — COMMENTED** [review]
How much of the list is shared between the two branches of the `if`? Maybe we can store the commands in a static `QStringList`?

File: client/chatline.cpp (hunk 1 above)
Comment (jwrober) [reply client/chatline.cpp:580]: Interesting. How so? It was the only header like it in the file

**Comment by jwrober** [comment]
> How much of the list is shared between the two branches of the `if`? Maybe we can store the commands in a static `QStringList`?

Actually a lot of it is the same. Let me look into a static list defined first.

File: client/chatline.cpp (hunk 1 above)
Comment (lmoureaux) [reply client/chatline.cpp:580]: https://longturn.readthedocs.io/en/latest/Coding/guidelines.html#documentation-comments

Comment (jwrober) [reply client/chatline.cpp:580]: Not consistent then. most of the function comments in the file use this convention

```
/**
   some comment
*/
```

Comment (lmoureaux) [reply client/chatline.cpp:580]: Yes, most functions are still like this because I didn't dare mass-replacing the old style.

**Pull request merged**